subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -1120,6 +1120,71 @@
             "changes": [
                 "Method has been removed"
             ]
+        },
+
+        {
+            "type": "org.gradle.plugin.devel.tasks.ValidateTaskProperties",
```

Comment:
Strange, I though we allow the removal of deprecated classes without the need to add an entry to the api-changes.json. And if we remove the class, I would expect the methods also count as removed.

Reply:
Well that's not the case. The checker would have to be modified to allow this (note that this isn't the first removal in this file).

Reply:
@jjohannes @blindpirate I vaguely remember this working before. Did something change?

Reply:
I don't remember anything like that. I remember that we always added removed classes/API here. Also when they were deprecated before.

I also think the current behavior is good. Only because something is deprecated does not mean it can be arbitrarily (or accidentally) removed at any time. Because usually we only want to remove things in major releases. And also then it should be a very conscious decision.

If we ever get to including the information from this file into the release notes (or some other place) this is also useful information.

Reply:
I don't think that's what Stefan is asking for: it's more the fact that if you remove class X, you shouldn't have to add all members of X to the file too (because it's enclosing).

Reply:
Mh ok. I actually looked at simplifying that the other day and it is not a simple change. So I am pretty sure it was always like it is now.